lib/src/native/realm_core.dart

```diff
@@ -61,25 +61,25 @@ class _RealmCore {
 
   String get libraryVersion => _realmLib.realm_get_library_version().cast<Utf8>().toDartString();
 
-  LastError? getLastError([Allocator? allocator]) {
-    if (allocator != null) {
-      final error = _realmLib.realm_get_last_error();
-
-      if (error == nullptr) {
-        return null;
-      }
-
-      String? message;
-      if (error.ref.message != nullptr) {
-        message = error.ref.message.cast<Utf8>().toDartString();
-      }
-      _realmLib.realm_release_last_error(error);
+  LastError? getLastError(Arena arena) {
```

Comment:
why explicit `Arena` here. `Allocator` is more generic.

Reply:
Fixed